Use YugabyteDB as the copyright name

jayant07-yb opened the pull request:
Using the corporate name (YugabyteDB, Inc) in the `yb-cpplint`.

Commit:
Changed the YUGABYTE_COPYRIGHT to '// Copyright (c) YugabyteDB, Inc.'

jasonyb commented on cpplint/cpplint.py:
```diff
@@ -1756,7 +1756,7 @@ def CheckForCopyright(filename, lines, error):
-  YUGABYTE_COPYRIGHT = "// Copyright (c) Yugabyte, Inc."
+  YUGABYTE_COPYRIGHT = "// Copyright (c) YugabyteDB, Inc."
   ALLOWED_COPYRIGHT_LINES = [YUGABYTE_COPYRIGHT,
                              "// Copyright (c) YugaByte, Inc.",
```
It looks like we have been allowing old copyrights, so I think you should do the same.

jasonyb replied:
```suggestion
                             "// Copyright (c) YugaByte, Inc.",
                             "// Copyright (c) Yugabyte, Inc.",
```

jayant07-yb committed:
Allowing old copyrights

Adding '"// Copyright (c) Yugabyte, Inc."' in ALLOWED_COPYRIGHT_LINES

Co-authored-by: jasonyb <93959687+jasonyb@users.noreply.github.com>

jasonyb commented on cpplint/cpplint.py:
```diff
@@ -1756,9 +1756,10 @@ def CheckForCopyright(filename, lines, error):
                              "// Copyright (c) YugaByte, Inc.",
+                             "// Copyright (c) Yugabyte, Inc.",
                              "// Portions Copyright (c) YugaByte, Inc.",
                              "// Portions Copyright (c) Yugabyte, Inc.",
```
```suggestion
                             "// Portions Copyright (c) Yugabyte, Inc.",
                             "// Portions Copyright (c) YugabyteDB, Inc.",
```

jasonyb reviewed (commented):
You could've looked at commit history to see what lines should change.  a24777775695b4899229650ebd7455b055507f70

jayant07-yb committed:
Modifying ALLOWED_COPYRIGHT_LINES as per the corporate name

Modifying 'Portions Copyright (c) YugaByte, Inc.' to
'Portions Copyright (c) YugabyteDB, Inc.'

Co-authored-by: jasonyb <93959687+jasonyb@users.noreply.github.com>

jasonyb reviewed (approved):
Thanks for the quick changes.  Title could be shorter like the previous commit I mentioned.
